### DISCUSSION TITLE
How to rate limit when running from an on demand jenkins job?

### DISCUSSION THREAD

**Discussion by rvarun11** [renovatebot/renovate#23817] [Request Help] [answered]
### How are you running Renovate?

Self-hosted

### If you're self-hosting Renovate, tell us what version of Renovate you run.

latest

### If you're self-hosting Renovate, select which platform you are using.

None

### Was this something which used to work for you, and then stopped?

I never saw this working

### Wanted end result.

We have an on demand Jenkins job which runs runs Renovate once every day at a set time. We want to update our Renovate config such that it creates 10 PRs on every run each repository. This is to so that our CI doesn't crash.

### What you tried so far.

We have tried using `prHourlyLimit: 10` but every run ends up creating all the PRs possible. 

Our config file is simple:
```js
{
 prHourlyLimit: 10
}
```

What is the expected outcome of this configuration? How does `prHourlyLimit` and `prConcurrentLimit` work when running from an on-demand job once per day.

**Comment by rarkins** [comment, marked as answer]
The prHourlyLimit should work. Please test against one repo and provide the debug logs of it _not_ working.

**Comment by rvarun11** [comment]
That's what I thought too. I was testing `prHourlyLimit` on the Gerrit PR and wanted to confirm if it's failing on Gerrit only. I have mentioned this https://github.com/renovatebot/renovate/pull/18961#issuecomment-1678170589
